fastchat/serve/gradio_web_server.py

```diff
@@ -140,27 +139,21 @@ def get_model_list(
         )
         models += list(openai_compatible_models_info.keys())
 
-    if add_chatgpt:
-        models += [
-            "gpt-4-0314",
-            "gpt-4-0613",
-            "gpt-3.5-turbo-0613",
-            "gpt-3.5-turbo-1106",
-        ]
-    if add_claude:
-        models += ["claude-2.1", "claude-2.0", "claude-instant-1"]
-    if add_palm:
-        models += ["gemini-pro"]
     models = list(set(models))
-
-    hidden_models = ["gpt-4-0314", "gpt-4-0613"]
-    for hm in hidden_models:
-        del models[models.index(hm)]
+    visible_models = models.copy()
+    for mdl in visible_models:
+        if mdl not in openai_compatible_models_info:
```

Comment:
1. Document the format of this file. https://github.com/lm-sys/FastChat/blob/1368f3fca0a4d5c55831325fd652fc650a52ba78/fastchat/serve/gradio_web_server.py#L76-L85
2. It is not openai-only anymore. Should we rename it to something like `api_models_info`?

Reply:
@Ying1123 Good point! Just rename and update the docstring. does it look good?

Reply:
Yes, thanks!